### ISSUE TITLE
Unexpected token export when library is included

### ISSUE THREAD

**Issue by David-Neto** [deblanco/xlsExport#1]
I'm including the library into my project from a folder using:
"<script type="text/javascript" src="lib/xlsExport-master/xls-export.js"></script>"

I get the following error pointing to the export line (line 95) with following message:
"Uncaught SyntaxError: Unexpected token export"

jsfiddle isn't working and throws similar error

**Comment by David-Neto** [comment]
Ok, I found what I was doing wrong. The library is written using ES6 and to work as is, it has to be compiled using Babel. 

To use it I've transpiled the contents of xls-export.js using [(https://babeljs.io/repl/)]. Notice it's necessary to exclude the last line: `export default XLSexport;` 
Also important is to use 'var xls = new XlsExport(data_example, 'name_example');' instead of 'const xls = new XlsExport(data, 'Example WB');' for instantiation as 'const' is also ES6 syntax.
